pipelines/openshift-ipi-deployment/0.2/openshift-ipi-deployment.yml

```diff
@@ -0,0 +1,60 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: openshift-ipi-deployment-v0-2
+spec:
+  params:
+  - default: create
+    description: Type of operation to perform, use 'create' for cluster creation or 'delete' for cluster deletion.
+    name: CLUSTER_OPERATION
+    type: string
+  - description: |-
+      Image containing openshift-install binary
+      eg. quay.io/openshift/openshift-installer
+    name: INSTALLER_IMAGE
+    type: string
+
+  tasks:
+  - name: install-openshift
+    params:
+    - name: OPERATION
+      value: $(params.CLUSTER_OPERATION)
+    - name: IMAGE
+      value: $(params.INSTALLER_IMAGE)
+    taskRef:
+      kind: Task
+      name: openshift-ipi-install-v0-2
+    workspaces:
+    - name: persistent
+      workspace: persistent-data
+    - name: config
+      workspace: config-map
+    - name: auth
+      workspace: secret
+  - name: wait-for-api
+    params:
+    - name: SCRIPT
+      value: while ! curl https://api.openshift-demo.cor00005-2.cna.ukcloud.uk:6443
```

Comment:
Should this hard-coded url be a parameter instead?

Reply:
Probably a parameter - good point